docs/user/exam_mode.rst

```diff
@@ -1,9 +1,11 @@
-Exam Mode
-=========
+.. _exams:
+
+Exams
+=====
 
 Artemis now includes an online exam mode:
 
 .. toctree::
 
    exams/instructors_guide
-   exams/students_guide
\ No newline at end of file
+   exams/students_guide
```

Comment:
I'm trying to spot the change here but just can't find it... What is the reason for this?